### PR TITLE
Extend timeout for berkeley connection test

### PR DESCRIPTION
**Pull request by dkijania**
Alter connect to berkeley test by extending timeout a bit after which we fail test. Currently test fails due to timeout expiration and we don't really know if local node is compatible with ITN or not

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by ghost-not-in-the-shell — APPROVED** [review]

File: buildkite/scripts/connect-to-berkeley.sh [hunk 1]
```diff
@@ -43,10 +43,10 @@ mina daemon \
   --libp2p-keypair "/root/libp2p-keys/key" \
 & # -background
 
-# Attempt to connect to the GraphQL client every 10s for up to 4 minutes
```
Comment (ghost-not-in-the-shell) [inline buildkite/scripts/connect-to-berkeley.sh:46]: nit: every 20s for up to 8 minutes

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Pull request merged**